packages/components/app/styles/components/tooltip.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TOOLTIP & TOOLTIP-BUTTON
+//
+
+@use "../mixins/focus-ring" as *;
+
+.hds-tooltip-button {
+  @include hds-focus-ring-with-pseudo-element(
+    $top: var(--token-tooltip-focus-offset),
+    $right: var(--token-tooltip-focus-offset),
+    $bottom: var(--token-tooltip-focus-offset),
+    $left: var(--token-tooltip-focus-offset),
+  );
+  margin: 0;
+  padding: 0;
+  background: transparent;
+  border: none;
+}
+
+// Prevent accidental override of hds classes ( :where() has a specificity of 0 )
```

Comment:
See "Font Inheritance" section added to the Showcase: https://hds-showcase-git-hds-1743-tooltip-component-hashicorp.vercel.app/components/tooltip

@alex-ju I worked with Cristiano to add the below styles to mitigate issues with font inheritance. Let me know if you have any feedback, thanks.

Reply:
Had a sync with Cristiano on this – I think this is a good usecase for the `:where` pseudoselector and a smart solution for this issue 🙌